content/source/docs/extend/testing/unit-testing.html.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: "extend-testing"
+page_title: "Extending Terraform - Unit Testing"
+sidebar_current: "docs-extend-testing-unit"
+description: |-
+  Extending Terraform is a section for content dedicated to developing Plugins
+  to extend Terraform's core offering.
+---
+
+# Unit Testing
+
+Testing plugin code in small, isolated units is distinct from Acceptance Tests,
+and does not require network connections. Unit tests are commonly used for
+testing helper methods that expand or flatten API responses into data structures
+for storage into state by Terraform. This section covers the specifics of
+writing Unit Tests for Terraform Plugin code.
+
+The procedure for writing unit tests for Terraform follows the same setup and
+conventions of writing any Go unit tests. We recommend naming tests to follow
+the same convention as our acceptance tests, `Test<Provider>_<Test Name>`. For more
+information on Go tests, see the [official Golang docs on testing][0]. 
+
+
+Below is an example unit test used in flattening AWS security group rules,
+demonstrating a typical `flattener` type method that's commonly used to convert
+structures returned from APIs into data structures used by Terraform in saving
+to state. The blow example is truncated, but you can see the full test in the
```

Comment:
below

Reply:
?

Reply:
Sorry that can be interpreted in the wrong context. Typo on line "The blow example".

Reply:
ahhhh, yes, that's a typo. Line 24 has Below as well, so I was confused 😄 

Fixed in 41545bb93